locale/en/docs/guides/blocking-vs-non-blocking.md

```diff
@@ -39,15 +39,15 @@ execute **asynchronously**.
 Using the File System module as an example, this is a **synchronous** file read:
 
 ```js
-const fs = require('fs');
-const data = fs.readFileSync('/file.md'); // blocks here until file is read
+const fs = require("fs");
```

Comment:
I think I most often see ''' used versus '"' so this change seems odd to me and unrelated to the translation?

Reply:
It was an unrelated change in https://github.com/nodejs/nodejs.org/pull/4999